src/webrtc/call.ts

```diff
@@ -1446,7 +1450,11 @@ export class MatrixCall extends EventEmitter {
 
         // party ID must match (our chosen partner hanging up the call) or be undefined (we haven't chosen
         // a partner yet but we're treating the hangup as a reject as per VoIP v0)
-        if (this.partyIdMatches(msg) || this.state === CallState.Ringing) {
+        if (
+            this.partyIdMatches(msg) ||
+            this.state === CallState.Ringing ||
+            this.state === CallState.InviteSent
```

Comment:
After thinking this over again, this probably doesn't make sense, does it?

Reply:
I wasn't thinking clearly. To explain this: we shouldn't end the call here because we want to give other devices a chance to answer the call